dashboard/modules/reporter/reporter_head.py

```diff
@@ -60,9 +62,11 @@ async def run(self, server):
 
         async for sender, msg in receiver.iter():
             try:
-                _, data = msg
+                key, data = msg
                 data = json.loads(ray.utils.decode(data))
-                DataSource.node_physical_stats[data["ip"]] = data
+                key = key.decode("utf-8")
+                node_id = key.split(":")[-1]
```

Comment:
Can you comment how key looks like here?

Reply:
OK. I will add some comments here.